.github/workflows/rc.yml

```diff
@@ -0,0 +1,136 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Release candidate verification
+
+on:
+  push:
+    paths:
+      - '.github/workflows/rc.yml'
+      # This is just a file we know will get edited when a version is bumped
+      # and shouldn't get touched otherwise
+      - 'ruby/red-arrow/lib/arrow/version.rb'
+  pull_request:
+    paths:
+      - '.github/workflows/rc.yml'
+      - 'ruby/red-arrow/lib/arrow/version.rb'
```

Comment:
Shouldn't this be a cron job instead? Or is the goal to run this _only_ when bumping the version?

Reply:
Probably so; this was my first time running these scripts so I wasn't sure how much of them depended on installing rc artifacts from a published repository, in which case we couldn't do them nightly unless we also published nightly artifacts (not a bad idea but out of scope).